README.md

```diff
@@ -1,2 +1,71 @@
 # docker-itchysats
-ItchySats in a Docker container
+
+A repo facilitating running ItchySats inside a [Docker][https://www.docker.com/what-docker] container on a x86-64 (e.g. macOS, Linux, Windows) or ARMv7 system.
+Docker website provides extensive tutorials on Docker installation, please refer there if you are new to Docker.
+
+The following guide assumes that you have Docker running, having [docker-compose][https://docs.docker.com/compose/install/] installed is also recommended.
```

Comment:
Lol. 
If you have to explain someone Docker in a readme about how to run ItchySats in a docker container  I don't think they will get far 😅 

I would prefer an introduction to ItchySats instead.

Reply:
well, this is a guide to running itchysats in a docker container - introduction to itchysats is linked at the bottom.
 I will add a link to `ItchySats`, so anyone confused can quickly jump there if they're confused (if they don't know, they should be on a different page :D)

Reply:
> well, this is a guide to running itchysats in a docker container - introduction to itchysats is linked at the bottom.

The question is, how do you expect people to find this repo? 

- If it pops up on Google, I'd expect to see an intro into ItchySats first. 
- If you link to it from a blog post/twitter thread explaining how to run ItchySats in a docker container, I would not expect anyone to click on the link who does not know Docker. 

Eitherway, I'm fine with having this here, it's just my two Sats :) 

> if they don't know, they should be on a different page :D

Totally agree with this :)

Reply:
thank you for the comments, I have reworked the readme to briefly describe ItchySats first and provide more links to the main repo. Please have a look it if looks more approachable.

as for Docker proficiency - I opted for the link so we would not exclude people who would like to try running it on their system without prior experience with Docker. it doesn't matter for the pros, they will just skip to the next sentence. :)

Reply:
thanks :)